torch/_dynamo/exc.py

```diff
@@ -226,10 +226,10 @@ def filter_stack(stack):
     return user_stack
 
 
-def format_error_msg(exc, code, record_filename=None, frame=None):
+def format_error_msg(exc, code, record_filename=None, frame=None, verbose=config.verbose):
```

Comment:
this will read `config.verbose` at program startup, but then ignore any changes to it after that.